src/steps/mdx-to-gridtables.ts

```diff
@@ -71,29 +71,18 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
       continue;
     }
 
-    // handle raw <hr /> conversion
-    if (node.name === 'hr') {
-      const wrapperNode = {
-        type: 'div',
-        children: [
-          {
-            type: 'text',
-            value: 'hr-wrapper'
-          },
-        ],
-      } as unknown as RootContent;
-
-      mdast.children.splice(i, 1, wrapperNode);
-      continue;
-    }
+    const blockName = node.name;
+
+    // If HorizontalLine, force slotsValue to 'none'
+    const isHorizontalLine = blockName === 'HorizontalLine';
 
     // get slots
     const slotsAttr = getAttribute(node, 'slots');
-    const slotsValue = getAttributeValue(slotsAttr, '');
-    // if (!slotsValue) {
-    //   // TODO: throw error for invalid document
-    //   break;
-    // }
+    let slotsValue = getAttributeValue(slotsAttr, '');
+    if (isHorizontalLine) {
```

Comment:
Don't add a slotsValue, just use isHorizontalLine throughout to check.  This is a safer way than checking if it's none and then checking the slot's size